_raml/apps/producttypes/examples/get_producttype_response.json

```diff
@@ -0,0 +1,104 @@
+{
+  "page": 1,
+  "items": [
+    {
+      "name": "Kleidung",
+      "productTypeId": "64180BCB-5AEF-4E69-4A92-AC150003B611",
+      "attributes": [
+        {
+          "visible": false,
+          "name": "Farbe",
+          "values": [
+            {
+              "value": "blau",
+              "name": "blau"
+            },
+            {
+              "value": "rot",
+              "name": "rot"
+            }
+          ],
+          "preDefAttribute": true
+        },
+        {
+          "visible": false,
+          "name": "Größe",
+          "values": [
+            {
+              "value": "S",
+              "name": "S"
+            },
+            {
+              "value": "M",
+              "name": "M"
+            },
+            {
+              "value": "L",
+              "name": "L"
+            }
+          ],
+          "preDefAttribute": true
+        }
+      ],
+      "alias": "Kleidung"
+    },
+    {
+      "name": "Schuhe",
```

Comment:
@mzeyen Another suitable product type might be "speaker" perhaps with the attribute "frequency response" or something like that?